src/model/element/entity.js

```diff
@@ -187,10 +187,10 @@ class Entity extends Element {
     };
```

Comment:
For the return, if we could start setting `id: assoc.dbPrefix` which is the MIRIAM standard, could make life easier, e.g. index.js `mapToUniprotIds`. I don't think this breaks anything?

Reply:
@jvwong would the updates in this commit (https://github.com/PathwayCommons/factoid/pull/904/commits/12c42a885a19d568c9c2345c153730055892d0b7) covers what you mean?